spec/api-debugger-spec.js

```diff
@@ -78,57 +83,62 @@ describe('debugger module', () => {
       }
     })
 
-    it('retuns response', (done) => {
+    it('returns response', done => {
       w.webContents.loadURL('about:blank')
       try {
         w.webContents.debugger.attach()
       } catch (err) {
-        return done('unexpected error : ' + err)
+        return done(`unexpected error : ${err}`)
       }
-      var callback = function (err, res) {
-        assert(!err.message)
-        assert(!res.wasThrown)
-        assert.equal(res.result.value, 6)
+
+      const callback = (err, res) => {
+        expect(err.message).to.not.exist()
```

Comment:
There shouldn't be a `message` property on `err`?
Or it should be there but have a value of `null` or `undefined`?

Reply:
it should be undefined :)